packages/terra-data-grid/src/index.js

```diff
@@ -1,4 +1,4 @@
 import WorklistDataGrid from './WorklistDataGrid';
+import FlowsheetDataGrid from './FlowsheetDataGrid';
 
-// eslint-disable-next-line import/prefer-default-export
```

Comment:
Do we still need this linter disable line?

Reply:
It isn't needed, since there are now multiple exports